apps/assistants/views.py

```diff
@@ -212,13 +213,23 @@ def delete(self, request, team_slug: str, pk: int):
             messages.success(request, "Assistant Archived")
             return HttpResponse()
         else:
+            version_query = None
+            if assistant.is_working_version:
+                version_query = list(
+                    map(
```

Comment:
I might be wrong, but I don't think we have to map the ids to strings?

Reply:
we do because the pipeline params stores the values as strings